src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -328,17 +184,20 @@ public void deleteNetworkModifications(UUID groupUuid, List<UUID> modificationsU
         }
     }
 
-    public void moveModifications(UUID groupUuid, UUID originGroupUuid, UUID before, UUID networkUuid, UUID reportUuid, UUID reporterId, String variantId, List<UUID> modificationsToMove, boolean canBuildNode) {
-        List<UUID> movedModifications = networkModificationRepository.moveModifications(groupUuid, originGroupUuid, modificationsToMove, before).getModificationsMoved();
-        if (canBuildNode && !movedModifications.isEmpty()) {
+    public void moveModifications(UUID groupUuid, UUID originGroupUuid, UUID before, NetworkInfos networkInfos, ReportInfos reportInfos, List<UUID> modificationsToMove, boolean canBuildNode) {
+        List<ModificationInfos> movedModifications = networkModificationRepository.moveModifications(groupUuid, originGroupUuid, modificationsToMove, before)
+            .stream()
+            .map(ModificationEntity::toModificationInfos)
+            .collect(Collectors.toList());
+        if (canBuildNode && !movedModifications.isEmpty() && networkInfos.isVariantPresent()) { // TODO remove canBuildNode and return NetworkDamages() ?
```

Comment:
TODO ... return NetworkDamages()
is it still necessary ? I thing the "damages" is already managed by the applicator class

Reply:
I think it will be necessary for later
it's to not forget